open-metadata-implementation/access-services/cognos/cognos-server/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project.  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>cognos</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>1.8-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <name>Cognos OMAS Server-side</name>
+    <description>
+        Server-side support for the Cognos Open Metadata Access Service (OMAS).
+    </description>
+
+    <artifactId>cognos-server</artifactId>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>admin-services-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>admin-services-registration</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>open-connector-framework</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>inmemory-repository-connector</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
```

Comment:
A -server component should not be dependent on any spring components.
We like to isolate these to our -spring components -- which are used in the server chassis & UI only - in part this is to make it easier for consumers to utilize their own rest api frameworks - and indeed to consider this for egeria itself in future

Reply:
Many server projects in access-services have this dependency.
Maven build for cognos-server fails without the dependency:

[WARNING] Used undeclared dependencies found:
[WARNING]    org.springframework:spring-core:jar:5.2.6.RELEASE:compile
[INFO] Add the following to your pom to correct the missing dependencies:
[INFO]
<dependency>
  <groupId>org.springframework</groupId>
  <artifactId>spring-core</artifactId>
  <version>5.2.6.RELEASE</version>
</dependency>

Reply:
The modules that have this dependency in their server module also have an issue raised against them.  You need to trace what is causing it and remove/change it.

Reply:
I didn't look thoroughly -- but I can see in src/main/java/org/odpi/openmetadata/accessservices/cognos/utils/QualifiedNameUtils.java you are using spring string utilities - so would be an easy fix to remove. There may be others

We have #2168 open to fix this for other modules

Reply:
fixed by https://github.com/odpi/egeria/pull/3044/commits/0e5b70c37236726ec3a327a879fd7b45d02a42ce